.ci/test_r_package.sh

```diff
@@ -222,8 +221,7 @@ fi
 if [[ $R_BUILD_TYPE == "cran" ]]; then
     pragma_warning_present=$(
         cat $BUILD_LOG_FILE \
-        | grep -E "warning: unknown pragma ignored" \
-        | wc -l
+        | grep --count -E "warning: unknown pragma ignored"
```

Comment:
Can we apply the same for
https://github.com/microsoft/LightGBM/blob/5dbd1df4b62a8fd33d93fe0a3c5f1bb9c3660e7c/.github/workflows/r_package.yml#L190
?

Reply:
oh yep!

Reply:
updated in https://github.com/microsoft/LightGBM/pull/3535/commits/cdabc9784234486563a41d2ce43aaf5c9521bf34